### ISSUE TITLE
Create new annotation zones for copied transcriptions

### ISSUE THREAD

**Issue by dleadbetter** [performant-software/textlab#240]

**Comment by dleadbetter** [comment]
@jamiefolsom - This change impacts the data model/underlying data. Should we also get the client's eyes on this before moving it to production to confirm the data still looks correct?

**Comment by jamiefolsom** [comment]
Good point. Sure -- I will ask Steve if he's got time to meet, unless there's a review app he can look at.

**Comment by dleadbetter** [comment]
All of the most recent changes have been deployed to https://mel-textlab.herokuapp.com: 

#232, #235, #214, #240 

I grabbed a backup of staging incase we need to restore the data for whatever reason.

**Comment by jamiefolsom** [comment]
OK, great -- thx.
